src/components/Card/types.ts

```diff
@@ -0,0 +1,48 @@
+import { ReactNode } from 'react';
+
+export interface CardProps {
+  /**
+   * 명함 속 이름 입니다.
+   */
+  name: string;
+  /**
+   * 명함 속 회사 입니다.
+   */
+  company: string;
+  /**
+   * 명함 속 직책 입니다.
+   */
+  position: string;
+  /**
+   * 명함 속 이메일 입니다.
+   */
+  email: string;
+  /**
+   * 명함 속 프로필컴포넌트 입니다.
+   */
+  profile?: ReactNode;
+  /**
+   * 명함 속 링크들의 정보 입니다.
+   */
+  linkInfo?: ReactNode;
+  /**
+   * 명함 속 명함생성이유 입니다.
+   */
+  reasons?: ReactNode;
+  /**
+   * 명함의 앞면과 뒷면의 상태를 정합니다.
+   */
+  isFront?: boolean;
+  /**
+   * 명함의 사이즈를 정합니다.
+   */
+  size?: 'default' | 'full';
+  /**
+   * 명함의 디자인를 정합니다.
+   */
+  designTemplate: string;
+  /**
+   * 명함의 스타일를 정합니다.
+   */
+  styleTemplate: string;
```

Comment:
```suggestion
  designTemplate: "city" | "default" | "future" ;
  /**
   * 명함의 스타일를 정합니다.
   */
  styleTemplate: number;
```
위처럼 타입을 구체화하는 편이 좋아보입니다. 스타일도 넘버로 받아서 png붙여주는 편이 안전할것같구용

Reply:
확인했습니다. 타입을 구체화하는게 제 생각에도 좋아보입니다. 수정하여 push 해놓겠습니다.
위에 카드 스타일은 어떻게 export하셨는지요? 지금 확인해봐도 다른 요소들과 같이 export하면 배경을 포함하여 같이만 되어서요😂

Reply:
아 그러네용,,, 제가 잘못본것같습니다. 따로 요청해서 받아야될 것 같네요

Reply:
추후 배열같은 곳에서 활용하려면 파일명도 1보다는 0부터 시작하는것이 좋아보입니다,

Reply:
넵! 일단은 타입 변경하고 작업이어서 나가겠습니다.